buku.py

```diff
@@ -2565,17 +2565,28 @@ def get_firefox_profile_name(path):
     profile : str
         Firefox profile name.
     """
-
-    try:
-        for name in os.listdir(path):
-            # can be in the format nnnnnn.default or nnnnnn.default-nnnnnn...
-            if '.default' in name and os.path.isdir(os.path.join(path, name)):
-                logdbg(name)
-                return name
-    except FileNotFoundError:
-        pass
-
-    return None
+    from configparser import ConfigParser, NoOptionError
+
+    profile_path = os.path.join(path, 'profiles.ini')
+    if os.path.exists(profile_path):
+        config = ConfigParser()
+        config.read(profile_path)
+        profiles_names = [section for section in config.sections() if section.startswith('Profile')]
+        if not profiles_names:
```

Comment:
Please test the cases below:

- profiles.ini is a blank file
- profiles.ini is a directory

Reply:
If profiles.ini blank:
```
./buku.py --ai
Add parent folder names as tags? (y/n): y
Import bookmarks from google chrome? (y/n): y
Import bookmarks from firefox? (y/n): y
Could not import bookmarks from firefox

Auto-generated tag: 2017Sep20
```

Reply:
if `profiles.ini` is a dir -- output the same

Reply:
OK.